examples/reference/publication_quality/bokeh/bar.py

```diff
@@ -0,0 +1,209 @@
+
+# ============================================================================
+# BOKEH THEME SETUP - Define global styling
+# ============================================================================
+
+ACCENT_COLOR = '#007ACC'  # Professional blue
+FONT_FAMILY = 'Roboto'
+
+def create_bokeh_theme(font_family=FONT_FAMILY, accent_color=ACCENT_COLOR):
+    """Create custom theme with specified font. Default: Roboto"""
+    return Theme(json={
+        'attrs': {
+            'Title': {
+                'text_font': font_family,
+                'text_font_size': '16pt',
+                'text_font_style': 'bold'
+            },
+            'Axis': {
+                'axis_label_text_font': font_family,
+                'axis_label_text_font_size': '12pt',
+                'axis_label_text_font_style': 'bold',
+                'major_label_text_font': font_family,
+                'major_label_text_font_size': '10pt',
+                'major_tick_line_color': "black",  # Remove tick marks
+                'minor_tick_line_color': None
+            },
+            'Plot': {
+                'background_fill_color': '#fafafa',
+                'border_fill_color': '#fafafa'
+            },
+            'Legend': {
+                'label_text_font': font_family,
+                'label_text_font_size': '10pt'
+            },
+            'Toolbar': {
+                "autohide": True,
+                "logo": None,
+                "stylesheets": [
+                    f"""
+                    .bk-OnOffButton.bk-active{{
+                        border-color: {accent_color} !important;
+                    }}
+                    """
+                ]
+            },
+            # Does not work via Theme, so added here for reference purposes until I figure out how to do it
+            'Tooltip': {
+                "stylesheets": [f"""
+                    .bk-tooltip-row-label {{
+                        color: {ACCENT_COLOR} !important;  
+            }}"""]
+                
+            }
+        }
+    })
+
+# Apply theme globally - affects all plots
+hv.renderer('bokeh').theme = create_bokeh_theme()
+
+# ============================================================================
+# HOLOVIEWS OPTS SETUP - Define global configuration
+# ============================================================================
+
+GLOBAL_BACKEND_OPTS={
+    'plot.xgrid.visible': False,           # Only horizontal grid lines
+    'plot.ygrid.visible': True,
+    'plot.ygrid.grid_line_color': "black",
+    'plot.ygrid.grid_line_alpha': 0.1,
+    'plot.min_border_left': 80,            # Add padding on left (for y-axis label)
+    'plot.min_border_bottom': 80,          # Add padding on bottom (for x-axis label)
+    'plot.min_border_right': 30,           # Add padding on right
+    'plot.min_border_top': 80,             # Add padding on top
+}
+
+ElementPlot.param.backend_opts.default = GLOBAL_BACKEND_OPTS
```

Comment:
I think this way of setting global opts is a hack. But it did not work setting them via `hv.opts.defaults`.